README.md

```diff
@@ -1,117 +1,121 @@
-   - `go get -u .`
-   - `go run language-analyzer.go` ,
-   or Python
-   - `pip install lookout-sdk`
-   - `python3 language-analyzer.py`
- - test **without** Github access, on the latest commit in some Git repository in local FS
-   ```
-   $ lookout-sdk review \
-     --log-level=debug \
-     --git-dir="$GOPATH/src/gopkg.in/src-d/lookout-sdk.v0" \
-     "ipv4://localhost:2020"
-   ```
-
-this will create a "mock" Review event and notify the analyzer, as if you were creating a Pull Request from `HEAD~1`.
-
-Check [src-d/lookout](https://github.com/src-d/lookout/tree/master/sdk#lookout-sdk-commands) for further details on `lookout-sdk` binary CLI options.
-
-
-Extracting UAST from arbitrary code
-===================================
-
-Lookout server and `lookout-sdk` binary proxy bblfsh gRPC protocol through DataServer.
-
-Please check how to create client in [bblfsh documentation](https://docs.sourced.tech/babelfish) and use DataServer address to connect.
-
-
-Caveats
-========
- - client: disable secure connection on dialing with `grpc.WithInsecure()`
- - client: set [max gRPC message size](https://github.com/grpc/grpc/issues/7927):
-    - go: use `pb.DialContext` instead.
-    - python: use `lookout.sdk.grpc.create_channel` instead.
- - client: turn off [gRPC fail-fast](https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md) mode
-   If your analyzer greedy creates a connection to DataServer before one was actually started, you might want to disable fail-fast mode. This way the RPCs are queued until the chanel ready. Here is an [example](https://github.com/src-d/lookout-gometalint-analyzer/blob/7b4b37fb3109299516fbb43017934d131784f49f/cmd/gometalint-analyzer/main.go#L66).
-  - client/server RFC 3986 URI scheme support:
-    - go: use `pb.ToGoGrpcAddress` and `pb.Listen`
-    - python: use `lookout.sdk.grpc.to_grpc_address`
-
-Release Process
-=================
- - Make sure the code is up to date using `make protogen`.
- - Update `VERSION` in `python/setup.py` with the same version that you will use for the tag (manual step required until [#2](https://github.com/src-d/lookout-sdk/issues/2) is implemented).
- - Create the release tag.
-
- # License
-[Apache License v2.0](./LICENSE)
+<p align="center">
+  <a href="https://github.com/src-d/lookout-sdk/releases">
+    <img src="https://badge.fury.io/gh/src-d%2Flookout-sdk.svg"
+         alt="GitHub version">
+  </a>
+  <a href="https://pypi.org/project/lookout-sdk/">
+    <img src="https://badge.fury.io/py/lookout-sdk.svg"
+         alt="PyPI version">
+  </a>
+  <a href="https://travis-ci.org/src-d/lookout-sdk">
+    <img src="https://travis-ci.org/src-d/lookout-sdk.svg?branch=master"
+         alt="Build Status">
+  </a>
+  <a href="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0/pb">
+    <img src="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg"
+         alt="GoDoc">
+  </a>
+  <a href="https://docs.google.com/document/d/1pqz-_SHO5BsJE-aa8o_bAY3r5vR67amnWN8-qZc2UgY">
```

Comment:
Why the link to the design doc? Did you take this from some template/other project?

I don't really like it. Since it's at the same level as the godoc reference some people may follow it expecting reference material. The design doc was useful for the planning phase, but now we are not keeping it up to date with the latest decisions and features. Also, this particular doc is not even well formatted, I would not link it at the same level as the godoc reference... I would not link it at all.